buildlib/jucx/jucx-test.yml

```diff
@@ -32,16 +32,26 @@ jobs:
           if [ $res -ne 0 ]; then
             exit 0;
           fi
+          with_cuda=no
+          if [[ $AGENT_NAME == *"gpu"* ]]; then
```

Comment:
can we check GPU device presence?
for example https://github.com/openucx/ucx/blob/aae491b5dffe575e7bdb9ae3251c5305e88e1262/contrib/test_jenkins.sh#L150
and add this helper function to az-helpers.sh